src/compaction.c

```diff
@@ -30,6 +30,12 @@ typedef struct SingleValueContext
     char isResetted;
 } SingleValueContext;
 
+typedef struct LastValueContext
+{
+    double value;
+    double lastValue; // used for empty buckets calculation
```

Comment:
@OfirMos Im not sure I understand why we need 2 values, maybe its better to have a `bool` that signals if we had any value.